src/vs/workbench/contrib/extensions/browser/extensions.contribution.ts

```diff
@@ -501,7 +504,10 @@ class ExtensionsContributions extends Disposable implements IWorkbenchContributi
 				id: MenuId.CommandPalette,
 				when: CONTEXT_HAS_GALLERY
 			},
-			run: () => runAction(this.instantiationService.createInstance(SearchExtensionsAction, '@category:"programming languages" @sort:installs '))
+			run: async () => {
+				const recommended = await this.tasExperimentService.getTreatment<boolean>('recommendedLanguages');
```

Comment:
tasExperimentService is marked as an optional dependency so before using it, you should check that its different than `undefined`.